### DISCUSSION TITLE
1.0 Preview 3 Release is LIVE!

### DISCUSSION THREAD

**Discussion by dayman40** [microsoft/WindowsAppSDK#1662] [Announcements]
## 1.0 Preview 3 Release of Windows App SDK 

We've just released the latest version of 1.0 Preview 3 of the Windows App SDK. This is the first version that adds Windows 11 styles to WinUI 3, **and WinUI 3 unpackaged support!** This release includes new features, bug fixed, and provide access to APIs and features we are intending to include in the next 1.0 stable release that's coming out very soon!   

## What's new 
This release of WinUI 3 is focused on supporting unpackaged apps on x86 and x64. Please view [Instructions for WinUI 3 unpackaged desktop apps](https://aka.ms/winui3/unpackaged-apps) to configure your WinUI 3 application to support unpackaged deployment. Keep an eye for ARM64 support that will be added in the next stable release.

- If you're building an app with the Windows App SDK for the first time, follow the [getting started instructions here.](https://docs.microsoft.com/windows/apps/project-reunion/get-started-with-project-reunion) 

To see everything that's new and changed, see the full [release notes here](https://aka.ms/windowsappsdk/1.0-preview-3-release-notes). For installation instructions, see  [Set up your development environment](https://docs.microsoft.com/windows/apps/windows-app-sdk/set-up-your-development-environment).  

## What's next 
To keep up with progress being made on the Windows App SDK, please see our [feature roadmap](https://github.com/microsoft/ProjectReunion/blob/main/docs/roadmap.md), which gets updated regularly.  

<hr /><em>This discussion was created from the release <a href='https://github.com/microsoft/WindowsAppSDK/releases/tag/v1.0.0-preview3'>1.0 Preview 3 Release is LIVE! </a>.</em>

**Comment by shelllet** [comment]
Great!

**Comment by Marv51** [comment]
Please clarify in the release notes, what "we are aiming to fix this in the next stable release" means.

Do you mean "until the stable release" or "after the release of 1.0"?
